migration/sql-files/085-delete-system.number-json-field.sql

```diff
@@ -0,0 +1,2 @@
+-- This removes any potentially existing system.number field from all work items.
+UPDATE work_items SET fields=(fields - 'system.number');
```

Comment:
Text files are supposed to end with a new line the end of file.
In VS Code, you may add these settings, if not already:
```
    "files.trimTrailingWhitespace": true,
    "files.insertFinalNewline": true,
    "files.trimFinalNewlines": true,
```

Reply:
I'm not sure what the big deal is with this but here you go: 
8c9647d